test/configCases/wasm/sharedarraybuffer-import-object/module.wat

```diff
@@ -0,0 +1,5 @@
+(module
+  (import "js" "mem" (memory 3 4 shared))
```

Comment:
The test failures seem to be related to the `shared` flag in `WebAssembly.Memory`. In Node.js 10.24.1, the `--experimental-wasm-threads` runtime flag is required to enable this feature. However, in Node.js 18.13.0, this flag has been removed, making it impossible to test across both versions simultaneously.<br/>
Could you advise on how to handle this scenario?